js/index.js

```diff
@@ -0,0 +1,119 @@
+let books = JSON.parse(localStorage.getItem('books'));
+
+//  Error Message
+function errorMsg(message) {
+  document.querySelector('.warning').textContent = message;
+  setTimeout(() => {
+    document.querySelector('.warning').textContent = '';
+  }, 2000);
+}
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  addBook() {
+    const { title, author, id } = this;
+    const bookList = { title, author, id };
```

Comment:
- [OPTIONAL] You have done very well implementing your project thus far. I think you should write descriptive variable names. `bookList` is a bit confusing if it refers to a book object as in this case. what's your take 🤔 ?

Reply:
understood, i will take that into account.